protos/sources/vega/data/v1/spec.proto

```diff
@@ -19,6 +19,10 @@ message PropertyKey {
   string name = 1;
   // type is the type of the property.
   Type type = 2;
+  // An optional decimal place to be be applied on the provided value
+  // valid only for PropertyType of type DECIMAL and INTEGER
+  uint32 number_decimal_places = 3;
```

Comment:
should this be optional

Reply:
I haven't set this as optional, because we take this value only if the type is integer or decimal.
If it is not set - will be 0 when deserialized.
Do you think its better to be set as optional here?

Reply:
optional would make sense, specifically because it is not mandatory to have it as you say?

Reply:
all right

Reply:
Set the fields to optional everywhere.